spec/features/ncr_work_orders_spec.rb

```diff
@@ -207,6 +207,16 @@
       expect(current_path).to eq("/proposals/#{work_order.proposal.id}")
     end
 
+    it "does not resave unchanged requests" do 
+      visit "/ncr/work_orders/#{work_order.id}/edit"
+      click_on 'Submit for approval'
+      visit "/ncr/work_orders/#{work_order.id}/edit"
+      click_on 'Submit for approval'
+
+      expect(current_path).to eq("/proposals/#{work_order.proposal.id}")
+      expect(page).to have_content("No changes were made to the request")
```

Comment:
Might be good to test that no emails were sent out, either. You should be able to use `deliveries` for this